[{"repo_name": "traccar/traccar", "id": "67", "repo_id": "107", "terms_vulnerability": "secur", "url": "https://github.com/traccar/traccar/pull/4512", "fix_commit_hash": null, "inducing_commit_hash_pyszz": ["9633eccdead886ed0b580707ad76cdf06df9a603"]}, {"repo_name": "traccar/traccar", "id": "68", "repo_id": "107", "terms_vulnerability": "cve", "url": "https://github.com/traccar/traccar/pull/4371", "fix_commit_hash": "1417443c6023f71a16d7b17549e2601c8cd05c2d", "inducing_commit_hash_pyszz": []}, {"repo_name": "traccar/traccar", "id": "74", "repo_id": "107", "terms_vulnerability": "secur", "url": "https://github.com/traccar/traccar/pull/3860", "fix_commit_hash": null, "inducing_commit_hash_pyszz": ["9633eccdead886ed0b580707ad76cdf06df9a603"]}, {"repo_name": "traccar/traccar", "id": "75", "repo_id": "107", "terms_vulnerability": "secur", "url": "https://github.com/traccar/traccar/pull/3724", "fix_commit_hash": null, "inducing_commit_hash_pyszz": ["9633eccdead886ed0b580707ad76cdf06df9a603"]}, {"repo_name": "traccar/traccar", "id": "90", "repo_id": "107", "terms_vulnerability": "secur", "url": "https://github.com/traccar/traccar/pull/2591", "fix_commit_hash": "5f867c6077f79a2a1d3b8ec18f78c3a2657ba698", "inducing_commit_hash_pyszz": []}, {"repo_name": "traccar/traccar", "id": "91", "repo_id": "107", "terms_vulnerability": "secur", "url": "https://github.com/traccar/traccar/pull/2552", "fix_commit_hash": "b0aa4698e95658f57ec2d3e0ed4f3fe0db8500a2", "inducing_commit_hash_pyszz": []}, {"repo_name": "traccar/traccar", "id": "94", "repo_id": "107", "terms_vulnerability": "secur", "url": "https://github.com/traccar/traccar/pull/1713", "fix_commit_hash": null, "inducing_commit_hash_pyszz": ["9633eccdead886ed0b580707ad76cdf06df9a603"]}, {"repo_name": "traccar/traccar", "id": "98", "repo_id": "107", "terms_vulnerability": "secur", "url": "https://github.com/traccar/traccar/pull/1252", "fix_commit_hash": "771e2d7c4ceb34c0b62852130061b04640b8ee71", "inducing_commit_hash_pyszz": []}]